ts/screens/modal/IdentificationModal.tsx

```diff
@@ -424,11 +425,15 @@ class IdentificationModal extends React.PureComponent<Props, State> {
           accessible={true}
           ref={this.headerRef}
         >
-          {I18n.t(
-            isValidatingTask
-              ? "identification.titleValidation"
-              : "identification.title"
-          )}
+          {isValidatingTask && I18n.t("identification.titleValidation")}
+          {!isValidatingTask &&
+            this.props.profileName &&
+            `${I18n.t("identification.titleProfileName", {
+              profileName: this.props.profileName
+            })}`}
+          {!isValidatingTask &&
+            !this.props.profileName &&
+            I18n.t("identification.title")}
```

Comment:
Do you think that this can be a more synthetic alternative:
```suggestion
   {isValidatingTask
            ? I18n.t("identification.titleValidation")
            : this.props.profileName
            ? I18n.t("identification.titleProfileName", {
                profileName: this.props.profileName
              })
            : I18n.t("identification.title")}
```
In this way we avoid to repeat the same check 3 times.

Reply:
You're right.
In order to avoid the lint error, I tried to refactor in this way:
```suggestion
          {isValidatingTask
            ? I18n.t("identification.titleValidation")
            : fromNullable(this.props.profileName).fold(
                I18n.t("identification.title"),
                pN =>
                  I18n.t("identification.titleProfileName", {
                    profileName: pN
                  })
              )}
```
What do you think?

Reply:
You're right.
In order to avoid the lint error, I tried to refactor as follow:
```suggestion
          {isValidatingTask
            ? I18n.t("identification.titleValidation")
            : fromNullable(this.props.profileName).fold(
                I18n.t("identification.title"),
                pN =>
                  I18n.t("identification.titleProfileName", {
                    profileName: pN
                  })
              )}
```
What do you think?

Reply:
Thank you, good idea!